data/operators/power/line.json

```diff
@@ -6334,7 +6334,7 @@
         "pacific gas and electric company"
       ],
       "tags": {
-        "operator": "Pacific Gas and Electric",
+        "operator": "Pacific Gas and Electric Company",
```

Comment:
PG&E is mentioned in two other places in NSI, so this should change those, too.

https://github.com/osmlab/name-suggestion-index/blob/2cf9fbda54a4c9cd37ed627424ed6dfe64b67266/data/operators/man_made/pipeline.json#L6124

https://github.com/osmlab/name-suggestion-index/blob/2cf9fbda54a4c9cd37ed627424ed6dfe64b67266/data/operators/power/plant.json#L2551